fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/spec/R4JDBCExamplesTest.java

```diff
@@ -40,16 +38,13 @@ public R4JDBCExamplesTest() throws Exception {
         this.properties = TestUtil.readTestProperties("test.jdbc.properties");
     }
 
-    @BeforeSuite
-    public void bootstrapAndLoad() {
-        System.out.println("Bootstrapping database:");
-
-
-        System.out.println("Processing examples:");
-    }
```

Comment:
intesting,  why this bootstrapAndLoad function was basically do nothing...

Reply:
I am not sure